source/common/upstream/upstream_impl.h

```diff
@@ -815,5 +819,10 @@ class BaseDynamicClusterImpl : public ClusterImplBase {
  */
 Network::DnsLookupFamily getDnsLookupFamilyFromCluster(const envoy::api::v2::Cluster& cluster);
 
+/**
+ * Utility function to get Dns from cluster.
+ */
+Network::DnsLookupFamily getDnsLookupFamilyFromCluster(const envoy::api::v2::Cluster& cluster);
```

Comment:
declared twice on lines 820 and 825?

Reply:
It's a merge error, thanks for pointing it out.